broker/fragment/store_azure.go

```diff
@@ -325,10 +325,18 @@ func (a *azureBackend) buildBlobURL(cfg AzureStoreConfig, client pipeline.Pipeli
 
```

Comment:
While we're in here, I've noticed that the stamped URLs have an extra '/' which I believe is due to this formatting in url.Parse ☝️ , right? Can we remove the extra "/"?